config/locales/oc.yml

```diff
@@ -342,13 +352,13 @@ oc:
       subject: "%{name} a mes vòstre estatut en favorit"
     follow:
       body: "%{name} vos sèc ara !"
-      subject: "%{name} es a vos sègre ara"
+      subject: "%{name} vos sèc ara"
     follow_request:
-      body: "%{name} a demandar a vos sègre"
+      body: "%{name} a demandat a vos sègre"
       subject: 'Demanda d’abonament : %{name}'
     mention:
-      body: 'Sètz estat mencionat per %{name} dins :'
-      subject: Sètz estat mencionat per %{name}
+      body: '%{name} vos a mencionat dins :'
+      subject: %{name} vos a mencionat
```

Comment:
This translation must be quoted because the string starts with percent character.